device/celestica/x86_64-cel_seastone-r0/sonic_platform/psu.py

```diff
@@ -71,11 +77,37 @@ def __get_gpio_base(self):
 
     def __get_gpio_value(self, pinnum):
         gpio_base = self.dx010_psu_gpio[0]['base']
-        gpio_dir = GPIO_DIR + '/gpio' + str(gpio_base+pinnum)
+        gpio_dir = GPIO_DIR + '/gpio' + str(gpio_base + pinnum)
         gpio_file = gpio_dir + "/value"
         retval = self._api_helper.read_txt_file(gpio_file)
         return retval.rstrip('\r\n')
 
+    def read_fru(self, path, attr_type):
+        content = []
+        attr_idx = 0
+        attr_length = 0
+
+        if(os.path.exists(path)):
+            with open(path, 'r', encoding='unicode_escape') as f:
+                content = f.read()
```

Comment:
Can you use the api_helper function to read the files?

Reply:
better not,   I need add " encoding='unicode_escape' " when use open,  but in api_helper functions , it don't have it,  if add it , i don't know if it will cause other functions